hbase-client/src/test/java/org/apache/hadoop/hbase/client/TestOnlineLogRecord.java

```diff
@@ -44,21 +44,21 @@ public void itSerializesScan() {
     Scan scan = new Scan();
     scan.withStartRow(Bytes.toBytes(123));
     scan.withStopRow(Bytes.toBytes(456));
-    String expectedOutput =
-      "{\n" + "  \"startTime\": 1,\n" + "  \"processingTime\": 2,\n" + "  \"queueTime\": 3,\n"
```

Comment:
it seems like adding a field completely changed the order of the serialized fields. all I did her was add the queryMetricsEnabled field